### DISCUSSION TITLE
using --no-sync-snap now that we have --exclude-snaps?

### DISCUSSION THREAD

**Discussion by secabeen** [jimsalterjrs/sanoid#890] [General]
In the past, when doing multi-replication setups, the common recommendation was use `--no-sync-snap` to prevent sync snaps from other replication jobs from showing up on the destination(s).  Now that we have `--exclude-snaps` in master, would it be a better solution in multi-replication setups to use sync snaps, then exclude them with `--exclude-snaps=syncoid`?

**Comment by phreaker0** [comment]
I don't quite understand, if you exclude the snapshots created by syncoid for replication than the don't serve a purpose and you won't need them anyway?

**Reply by secabeen** [reply]
You're excluding the sync-snaps made by another syncoid run.  If you have a Server S, with sync targets T and U, with Sync Snaps, syncoid will create two sync snaps: "syncoid-T-DATE" and "syncoid-U-DATE".   Each run, syncoid will send the sync-snap for the other target over, where it will then never get deleted.  So target T will have a bunch of "syncoid-U-DATE" snapshots, and vice versa.  Excluding the syncoid snaps doesn't prevent each run from using its own sync-snaps, just stops the syncing of the other ones.
